### PR TITLE
Use dynamic badge for latest release.

### PR DESCRIPTION
**Pull request by soufianenassih**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# GraalVM Dev Build Downloads
+# GraalVM Dev Build Downloads [![Latest dev release](https://img.shields.io/github/v/release/graalvm/graalvm-ce-dev-builds?color=brightgreen&label=latest%20release)](https://github.com/graalvm/graalvm-ce-builds/releases/latest)
```
Comment (ezzarghili) [inline README.md]: Link https://github.com/graalvm/graalvm-ce-builds/releases/latest is incorrect
should be https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest

**Review by ezzarghili — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (soufianenassih) [reply README.md]: Nice catch @ezzarghili you are right, thanks.

Comment (soufianenassih) [reply README.md]: Done

**Review by ezzarghili — APPROVED** [review]
LGTM

**Pull request merged**